mav_offboard/MavOffbCtlType ctlType
---
bool success